dockerfiles/Dockerfile.binary-native

```diff
@@ -1,4 +1,6 @@
-FROM    golang:1.11.13-alpine
+ARG GO_VERSION=1.12.10
+
+FROM    golang:${GO_VERSION}-alpine
```

Comment:
We may need changes in docker-ce-packaging to take the GO_VERSION into account

Reply:
I believe this is sufficient: https://github.com/docker/docker-ce-packaging/pull/389